### ISSUE TITLE
Change Default Tween Plugin

### ISSUE THREAD

**Issue by Kleptine** [Demigiant/dotween#170]
I'd like to make all of my color animations interpolate through the LAB color space. But to do this I need to be able to set a new default plugin for the Color type (or else pass a custom plugin everywhere I use a tween in my code, and have to reimplement all of the DOColor functions). 

This would be a pretty easy feature based on the PluginsManager code. Happy to contribute if you'd take the PR.

**Comment by Demigiant** [comment]
Ahoy!

You can already create custom plugins :) There's an example [here](http://dotween.demigiant.com/documentation.php#customPlugins) with code comments that explain it better.

**Comment by Kleptine** [comment]
Right! So I've already done that, but I like it to automatically use my custom plugin by default for all DOColor transitions, where it currently uses ColorPlugin instead. (ie. I'd like to override the PluginsManager GetDefaultPlugin function).

**Comment by Demigiant** [comment]
Ah, sorry! It was veeeery late when I answered. Sure, go with the PR and I'll check it out :)
